navigation/screens/facility-reviews.tsx

```diff
@@ -1,18 +1,58 @@
 import React from "react";
-import { Button, Image, ScrollView, StyleSheet, Text, View } from "react-native";
+import {
+  Button,
+  Image,
+  ScrollView,
+  StyleSheet,
+  Text,
+  View,
+} from "react-native";
+import {
+  likeReview,
+  dislikeReview,
+  replyReview,
+} from "@/controllers/reviewController";
+import { useState } from "react";
 
 export default function FacilityReviewsScreen() {
+  /**like a review */
+  const [reviewID, setReviewID] = useState("");
+  const [username, setUsername] = useState("");
+  const [reply, setReply] = useState("");
```

Comment:
note for the future (you don't need to do this for this PR):

need to create a text box that changes this value. text box should open when you click the review button?